### ISSUE TITLE
issues with qiime2 2024.5 vrsion

### ISSUE THREAD

**Issue by EorgeKit** [picrust/q2-picrust2#19]
when trying to install in qiime2 2024.5 version I get the following error about a python conflict
```
conda install q2-picrust2=2023.2 -c conda-forge -c bioconda -c gavinmdouglas 
Channels:
 - conda-forge
 - bioconda
 - gavinmdouglas
 - r
Platform: linux-64
Collecting package metadata (repodata.json): done
Solving environment: \ warning  libmamba Added empty dependency for problem type SOLVER_RULE_UPDATE
failed

LibMambaUnsatisfiableError: Encountered problems while solving:
  - package q2-picrust2-2023.2-py38_0 requires picrust2 2.5.2.*, but none of the providers can be installed

Could not solve for environment specs
The following packages are incompatible
├─ pin-1 is installable and it requires
│  └─ python 3.9.* , which can be installed;
└─ q2-picrust2 2023.2**  is not installable because it requires
   └─ picrust2 2.5.2.* , which requires
      └─ python >=3.5,<3.9 , which conflicts with any installable versions previously reported.

```
Please advise

**Comment by ebervilla** [comment]
Hi 

I am having the same issue. Its there any recommendation to solve it?

best,
Eber

**Comment by gavinmdouglas** [comment]
Hi there,

I just released a version compatible with QIIME2 v2024.5 -- see here: https://library.qiime2.org/plugins/q2-picrust2/13/


Cheers,

Gavin
